datastore-v1-proto-client/src/main/java/com/google/datastore/v1/client/RemoteRpc.java

```diff
@@ -53,9 +53,18 @@ class RemoteRpc {
   private final HttpRequestInitializer initializer;
   private final String url;
   private final AtomicInteger rpcCount = new AtomicInteger(0);
-  // Not final - so it can be set/reset in Unittests
-  private static boolean enableE2EChecksum =
-      Boolean.parseBoolean(System.getenv("GOOGLE_CLOUD_DATASTORE_HTTP_ENABLE_E2E_CHECKSUM"));
```

Comment:
If a user has the old env var set then it will be ignored and request checksum will remain enabled (as it is enabled by default now) and response checksum will become disabled.

I can provide a transition so that if the old env var is set then both checksums are enabled. But I specifically decided not to do that because of two reasons:

1. No one has the old env var set (I have an ESF metric for that).
2. We don't want to encourage enabling response checksum verification yet until we have a metric for success/error rates and figured out all edge-cases.

Reply:
got it, thanks! I think as an extra precaution, we should still check if this env variable is set and simply log a warning to the user that they should migrate/update their code.

Reply:
okay, sounds good. Added the handling of previous env var (it will still work -- enabling/disabling both request and response checksums if only the old env var is provided) and a warning. Is the wording for the warning okay?

Reply:
sorry if I was unclear - I think it's fine to *only* warn that the env variable is set, and not actually use it. Feel free to change the logic and warning message accordingly (the wording lgtm)

Reply:
no, I decided to keep the behavior of the old env variable so that it won't be a breaking change at all.